src/rules/builtins.rs

```diff
@@ -22,17 +23,11 @@ impl RuleDefinition for FixTrunc {
         "Fix `trunc()` usage based on type"
     }
 
-    fn get_node(&self, root: &Root) -> Result<SyntaxNode, RuleError> {
-        root.procedure()
+    fn find(&self, root: &Root, ctx: &DboAnalyzeContext) -> Result<Vec<RuleMatch>, RuleError> {
```

Comment:
My suggestion is to rename this method to be consistent with how the iterator `find` works, which returns a single `Option` with an inner item, instead of a list of items.

Reply:
What other name would you choose in this case? Something like `filter`?

Reply:
Good point, will do! Regarding the empty `Vec`, you already recommended it once, I will tackle it soon-ish (#75).